components/eviltools/parse.go

```diff
@@ -150,16 +150,16 @@ func parseCreateAccountParams(paramsAccountCreate *accountwallet.ParametersAccou
 		return nil, ierrors.New("implicit account cannot be created without Block Issuance Feature")
 	}
 
-	if !paramsAccountCreate.Implicit && !paramsAccountCreate.NoTransition {
+	if !paramsAccountCreate.Implicit && !paramsAccountCreate.Transition {
```

Comment:
NoTransition was to have transitioned account as a default behavior, now you always need to remember to add the flag. But if is was confusing, we can have it this way